src/App.js

```diff
@@ -1,31 +1,50 @@
 import React from 'react';
 import './App.css';
 import { RemixClient } from './RemixClient'
-
+const p = new RemixClient()
 function App() {
-  const p = new RemixClient()
+
   const openModal = () => {
     p.onConnect()
   }
+
+  const disconnect = () => {
+    p.onDisconnect()
+  }
+
+  const showButtons = (connected = true) =>{
+    document.getElementById("disconnectbtn").style.display = document.getElementById("accounts-container").style.display = connected? 'block':'none';
+    document.getElementById("connectbtn").style.display =  connected? 'none':'block';
+  }
+
   p.internalEvents.on('accountsChanged', (accounts) => {
-    document.getElementById('accounts').innerHTML = JSON.stringify(accounts)
+    document.getElementById('accounts').innerHTML = ""
+    for(const account of accounts){
+      document.getElementById('accounts').innerHTML += `<li className="list-group-item">${account}</li>`
+    }
+    
   })
 
   p.internalEvents.on('chainChanged', (chain) => {
     document.getElementById('chain').innerHTML = chain
+    showButtons(chain)
```

Comment:
`showButtons` is declared with `const showButtons = (connected = true) =>{`
`chain` doens't seem to be a boolean